### PR TITLE
[14.0][mig]agreement_project

### PR DESCRIPTION
**Pull request by hkapatel-initos**
Remove the deprecated decorators and field Module Migration to 14.0.

Dependencies:

agreement_legal: #669

### CODE REVIEW

**Comment by hkapatel-initos** [comment]
@patrickrwilson  can u please review this PR?

**Comment by hkapatel-initos** [comment]
@osi-scampbell and @kittiu can you please review this pr?

**Comment by hkapatel-initos** [comment]
Hi @ALL It would be great if someone could add a review to this PR.

File: agreement_project/views/agreement_view.xml
Comment (dsolanki-initos) [inline agreement_project/views/agreement_view.xml]: I think we dont require binding_view_types please check and remove it.

Comment (hkapatel-initos) [reply agreement_project/views/agreement_view.xml]: @dsolanki-initos I have done this can you please review again.

**Review by dsolanki-initos — APPROVED** [review]
LGTM

**Comment by patrickrwilson** [comment]
@hkapatel-initos can you increase code coverage to make all green? Looks like just 2 lines aren't getting hit.

**Comment by hkapatel-initos** [comment]
@Marcel Savegnago It would be nice if could you please review the PR?

**Comment by hkapatel-initos** [comment]
@ygol  Goldberg, can you please review this pr?

**Review by ygol — APPROVED** [review]
LGTM

**Comment by hkapatel-initos** [comment]
> @hkapatel-initos can you increase code coverage to make all green? Looks like just 2 lines aren't getting hit.

a suggestion is implemented can you please check again?

**Comment by patrickrwilson** [comment]
@hkapatel-initos can you squash all the automated commits (translations, readme upd, .pot). Then I think this can be merged after that history is cleaned up. Thanks

**Comment by hkapatel-initos** [comment]
> @hkapatel-initos can you squash all the automated commits (translations, readme upd, .pot). Then I think this can be merged after that history is cleaned up. Thanks

hello there are only 2 commits one is pre-commit and the second is Migration to v14

**Comment by patrickrwilson** [comment]
> > @hkapatel-initos can you squash all the automated commits (translations, readme upd, .pot). Then I think this can be merged after that history is cleaned up. Thanks
> 
> hello there are only 2 commits one is pre-commit and the second is Migration to v14

In the commit history includes contains 30 of them, which include all the automated ones from previous versions. For the migration process OCA wants those automated bot commits cleaned out with each migration. Please review the migration documents here: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

And the specific is: Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests for a procedure for reducing commits from "OCA Transbot...".

**Comment by hkapatel-initos** [comment]
> > > @hkapatel-initos can you squash all the automated commits (translations, readme upd, .pot). Then I think this can be merged after that history is cleaned up. Thanks
> > 
> > 
> > hello there are only 2 commits one is pre-commit and the second is Migration to v14
> 
> In the commit history includes contains 30 of them, which include all the automated ones from previous versions. For the migration process OCA wants those automated bot commits cleaned out with each migration. Please review the migration documents here: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> 
> And the specific is: Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests for a procedure for reducing commits from "OCA Transbot...".

**Comment by hkapatel-initos** [comment]
Hello, @patrickrwilson 
Here is the reference of your commit and contains histories from previous versions on agreement legal: #669,  so I think it's not necessary to squash all the automated commits from previous versions. ​ Thanks

**Comment by hkapatel-initos** [comment]
@pedrobaeza  It would be great if you could try to merge this PR.

**Comment by pedrobaeza** [comment]
I'm not qualified to review this PR and a PSC of this repo/maintainer of this module should review it.

**Comment by hkapatel-initos** [comment]
Hello maintainers,
Can you please try to merge it?

**Comment by hkapatel-initos** [comment]
Hello maintainers,
It would be nice if you are trying to merge it

**Comment by hkapatel-initos** [comment]
@dreispt  It would be great if you could try to merge this PR.

**Pull request merged**
